tests/functions/default_arguments.phpt

```diff
@@ -36,9 +36,9 @@ f1(10, 20, 30, 40, 50, 60);
 f1(1, 2, 3, 4, 5, 6, 7);
 //*/
 ///*
-// 2 default followed by one non-default; unusual, but permitted
+// 1 non-default followed by two default; unusual, but permitted
 
-function f2($p1 = 100, $p2 = 1.23, $p3)
+function f2($p1, $p2 = 1.23, $p3 = "abc")
```

Comment:
This was originally f4 and I renamed it to f2 to keep a sequence. The original f2 and f3 would also produce fatals because of the ArgumentCountException. Again, can be changed to catch them and keep the information.